cmd/glasskube/cmd/describe.go

```diff
@@ -203,14 +254,14 @@ func status(pkg *v1alpha1.Package) string {
 	if pkgStatus != nil {
 		switch pkgStatus.Status {
 		case string(condition.Ready):
-			return color.GreenString(pkgStatus.Status)
+			return pkgStatus.Status
 		case string(condition.Failed):
-			return color.RedString(pkgStatus.Status)
+			return pkgStatus.Status
 		default:
 			return pkgStatus.Status
 		}
 	} else {
-		return color.New(color.Faint).Sprint("Not installed")
+		return "Not installed"
```

Comment:
Removing the colored output here would be unfortunate. As an alternative, I would suggest not calling `status` from `createOutputStructure`, but rather setting the "status" property if `pkg != nil` like so:
```go
data["status"] = client.GetStatusOrPending(pkg).Status
```

It's OK if the status is only set if the package is installed, but optionally you can also set it to "not installed" in the initial map.

Reply:
Don't we need to see the `status` even if the Package is not installed?

Reply:
I think it would be fair to say that a missing status field would imply that a package is not installed. But as I said, you can also set the status to "not installed" in that case.